README.md

```diff
@@ -67,6 +67,25 @@ export const ExampleComponent = () => {
 
 </details>
 
+## Installation
+
+<details>
```

Comment:
I actually wanted two `details` section. One only for npm and one for yarn.

Should look like [this gist](https://gist.github.com/igeligel/7fb12f8808c15f777d66e03df6a074cd).